builder/lxc/step_provision.go

```diff
@@ -2,9 +2,10 @@ package lxc
 
 import (
 	"context"
-	"github.com/hashicorp/packer/common"
 	"log"
 
+	"github.com/hashicorp/packer/common"
+
```

Comment:
Why it keeps moving `hashicorp/packer` imports to a separated block from current `hashicorp/packer` import block 👀 ?

Reply:
Ah I think this is because goimports only splits other imports from the golib imports;  but doesn't know if things should be grouped. ( I also think they should )

Reply:
Yeah I need to see if goimports  can fix that or if we need to remove the extra blank line for things to get grouped together. Nice catch!

Reply:
Looks like this is a current limitation of goimports https://github.com/golang/go/issues/20818 But one we can address at a later time :smile: